src/wix/test/WixToolsetTest.Sdk/MsbuildFixture.cs

```diff
@@ -193,7 +193,7 @@ string ReplacePathsInMessage(string message)
             }
         }
 
-        [Theory(Skip = "Flaky")]
+        [Theory]
```

Comment:
I don't know why you reenabled this test in this PR. This test flakiness is due to https://github.com/wixtoolset/issues/issues/6720 and the associated MSBuild issue, not validation.

Reply:
You said the AppVeyor reproduced the issue. I re-enabled it to see if it reproduced. It didn't. If you still hit it intermittently due to not using the VS command prompt, do feel free to add the `Skip` back but be sure to include the link to the MSBuild issue since that has a good chance of living longer. I'm investigating wixtoolset/issues#6720 today.

Reply:
It worked because I changed Appveyor to temporarily use the VS command prompt: https://github.com/wixtoolset/wix4/pull/113/files#diff-92ab9a36df5d8e9f7076f2fdec59492d1ac2d9cf27ea046767a7fc4d542ef3dc. I expect it to start failing for everyone when 6720 is fixed.

Reply:
Awesome! Good to know I have a test already available to tell me when I have 6720 fixed. 👍🏻

Reply:
Well, yes. This test should have been failing this whole time. I'm pretty sure it was working properly when I set it up in the Tools micro repo, but either I didn't verify that it was actually running the 32-bit and 64-bit tasks or the MSBuild issue was introduced at the same time that we switched to the new repo.